manifests/nfs-common.pp

```diff
@@ -0,0 +1,111 @@
+# Class: cloudstack::nfs-common
```

Comment:
please rename to nfs_common
newer puppet version will complain when having classes with hyphens.